NetKAN/Mk2Expansion.netkan

```diff
@@ -13,6 +13,10 @@
     "suggests": [
 	{ "name": "FerramAerospaceResearch" }
     ],
+    "conflicts": [
```

Comment:
This creates an implicit conflict between `Mk2Expansion` and `StockalikeMiningExtension`.

`StockalikeMiningExtension` depends `InsanityAerospace` and `Mk2Expansion` conflicts `InsanityAerospace`.

Instead, remove this `conflicts` and `provides` and instead `depends InsanityAerospace` and filter the Agencies folder (as in `StockalikeMiningExtension`).